manifests/1.1.1/opensearch-1.1.1.yml

```diff
@@ -0,0 +1,17 @@
+---
+schema-version: "1.0"
+build:
+  name: OpenSearch
+  version: 1.1.1
+  patches: 
+    - 1.1.0
+components:
+  - name: OpenSearch
+    dist: https://ci.opensearch.org/ci/dbc/builds/1.1.0/405
```

Comment:
The path will need to be inferred/discovered which will have been broken with the recent url restructuring that only impacted the 1.2.0+ builds.  Should we add parameters to help the build system construct the correct path to the manifest?

Reply:
Either in a separate PR or maybe we should just copy the manifest with bad URLs to a corrected copy in S3 and use that instead.

Reply:
The manifest is oriented around the components of with a platform / architecture, do we need another kind of manifest that is oriented around a releases artifacts.  Note, we can move this out to another PR

Reply:
I don't see this as necessary. This is a bundle (assembly/distribution) manifest that can now include references to another bundle, and picks up components with matching platform/architecture.